Description: Prefix all binaries with topcom-.
 Several of the binaries (esp. cross and cube) have very generic names
 which may clash with other packages on the user's system.  Other distros
 have taken a similar approach (e.g., Gentoo renames cube to topcom_cube
 and Fedora prefixes cross, cube, cyclic, hypersimplex, and lattice with
 TOPCOM-).
 We take the same approach as the nauty package -- prefix *everything*.
 This has the advantage of consistency, which is useful for other programs
 which may call TOPCOM, e.g., Macaulay2's Topcom package.
Author: Doug Torrance <dtorrance@debian.org>
Forwarded: not-needed
Last-Update: 2022-06-23

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,125 +1,125 @@
 bin_PROGRAMS = \
-B_A \
-B_A_center \
-B_D \
-B_D_center \
-B_S \
-B_S_center \
-binomial \
-check \
-chiro2allfinetriangs \
-chiro2alltriangs \
-chiro2circuits \
-chiro2cocircuits \
-chiro2dual \
-chiro2finetriang \
-chiro2finetriangs \
-chiro2mintriang \
-chiro2nallfinetriangs \
-chiro2nalltriangs \
-chiro2ncircuits \
-chiro2ncocircuits \
-chiro2nfinetriangs \
-chiro2ntriangs \
-chiro2placingtriang \
-chiro2triangs \
-cocircuits2facets \
-cross \
-cube \
-cyclic \
-Dnxk \
-hypersimplex \
-lattice \
-kDn \
-permutahedron \
-points2allfinetriangs \
-points2alltriangs \
-points2chiro \
-points2circuits \
-points2cocircuits \
-points2facets \
-points2finetriang \
-points2finetriangs \
-points2flips \
-points2gale \
-points2mintriang \
-points2nallfinetriangs \
-points2nalltriangs \
-points2ncircuits \
-points2ncocircuits \
-points2nfinetriangs \
-points2nflips \
-points2ntriangs \
-points2placingtriang \
-points2prettyprint \
-points2symmetries \
-points2triangs \
-points2vertices \
-points2volume \
-santos_triang \
-santos_dim4_triang \
-santos_22_triang
+topcom-B_A \
+topcom-B_A_center \
+topcom-B_D \
+topcom-B_D_center \
+topcom-B_S \
+topcom-B_S_center \
+topcom-binomial \
+topcom-check \
+topcom-chiro2allfinetriangs \
+topcom-chiro2alltriangs \
+topcom-chiro2circuits \
+topcom-chiro2cocircuits \
+topcom-chiro2dual \
+topcom-chiro2finetriang \
+topcom-chiro2finetriangs \
+topcom-chiro2mintriang \
+topcom-chiro2nallfinetriangs \
+topcom-chiro2nalltriangs \
+topcom-chiro2ncircuits \
+topcom-chiro2ncocircuits \
+topcom-chiro2nfinetriangs \
+topcom-chiro2ntriangs \
+topcom-chiro2placingtriang \
+topcom-chiro2triangs \
+topcom-cocircuits2facets \
+topcom-cross \
+topcom-cube \
+topcom-cyclic \
+topcom-Dnxk \
+topcom-hypersimplex \
+topcom-lattice \
+topcom-kDn \
+topcom-permutahedron \
+topcom-points2allfinetriangs \
+topcom-points2alltriangs \
+topcom-points2chiro \
+topcom-points2circuits \
+topcom-points2cocircuits \
+topcom-points2facets \
+topcom-points2finetriang \
+topcom-points2finetriangs \
+topcom-points2flips \
+topcom-points2gale \
+topcom-points2mintriang \
+topcom-points2nallfinetriangs \
+topcom-points2nalltriangs \
+topcom-points2ncircuits \
+topcom-points2ncocircuits \
+topcom-points2nfinetriangs \
+topcom-points2nflips \
+topcom-points2ntriangs \
+topcom-points2placingtriang \
+topcom-points2prettyprint \
+topcom-points2symmetries \
+topcom-points2triangs \
+topcom-points2vertices \
+topcom-points2volume \
+topcom-santos_triang \
+topcom-santos_dim4_triang \
+topcom-santos_22_triang
 
-B_A_SOURCES                        = B_A.cc
-B_A_center_SOURCES                 = B_A_center.cc
-B_D_SOURCES                        = B_D.cc
-B_D_center_SOURCES                 = B_D_center.cc
-B_S_SOURCES                        = B_S.cc
-B_S_center_SOURCES                 = B_S_center.cc
-binomial_SOURCES                   = binomial.cc
-check_SOURCES                      = check.cc
-chiro2allfinetriangs_SOURCES       = chiro2allfinetriangs.cc
-chiro2alltriangs_SOURCES           = chiro2alltriangs.cc
-chiro2circuits_SOURCES             = chiro2circuits.cc
-chiro2cocircuits_SOURCES           = chiro2cocircuits.cc
-chiro2dual_SOURCES                 = chiro2dual.cc
-chiro2finetriang_SOURCES           = chiro2finetriang.cc
-chiro2finetriangs_SOURCES          = chiro2finetriangs.cc
-chiro2mintriang_SOURCES            = chiro2mintriang.cc
-chiro2nallfinetriangs_SOURCES      = chiro2nallfinetriangs.cc
-chiro2nalltriangs_SOURCES          = chiro2nalltriangs.cc
-chiro2ncircuits_SOURCES            = chiro2ncircuits.cc
-chiro2ncocircuits_SOURCES          = chiro2ncocircuits.cc
-chiro2nfinetriangs_SOURCES         = chiro2nfinetriangs.cc
-chiro2ntriangs_SOURCES             = chiro2ntriangs.cc
-chiro2placingtriang_SOURCES        = chiro2placingtriang.cc
-chiro2triangs_SOURCES              = chiro2triangs.cc
-cocircuits2facets_SOURCES          = cocircuits2facets.cc
-cross_SOURCES                      = cross.cc
-cube_SOURCES                       = cube.cc
-cyclic_SOURCES                     = cyclic.cc
-Dnxk_SOURCES                       = Dnxk.cc
-hypersimplex_SOURCES               = hypersimplex.cc
-lattice_SOURCES	                   = lattice.cc
-kDn_SOURCES                        = kDn.cc
-permutahedron_SOURCES              = permutahedron.cc
-points2allfinetriangs_SOURCES      = points2allfinetriangs.cc
-points2alltriangs_SOURCES          = points2alltriangs.cc
-points2chiro_SOURCES               = points2chiro.cc
-points2circuits_SOURCES            = points2circuits.cc
-points2cocircuits_SOURCES          = points2cocircuits.cc
-points2facets_SOURCES              = points2facets.cc
-points2finetriang_SOURCES          = points2finetriang.cc
-points2finetriangs_SOURCES         = points2finetriangs.cc
-points2flips_SOURCES               = points2flips.cc
-points2gale_SOURCES                = points2gale.cc
-points2mintriang_SOURCES           = points2mintriang.cc
-points2nallfinetriangs_SOURCES     = points2nallfinetriangs.cc
-points2nalltriangs_SOURCES         = points2nalltriangs.cc
-points2ncircuits_SOURCES           = points2ncircuits.cc
-points2ncocircuits_SOURCES         = points2ncocircuits.cc
-points2nfinetriangs_SOURCES        = points2nfinetriangs.cc
-points2nflips_SOURCES              = points2nflips.cc
-points2ntriangs_SOURCES            = points2ntriangs.cc
-points2placingtriang_SOURCES       = points2placingtriang.cc
-points2prettyprint_SOURCES         = points2prettyprint.cc
-points2symmetries_SOURCES          = points2symmetries.cc
-points2triangs_SOURCES             = points2triangs.cc
-points2vertices_SOURCES            = points2vertices.cc
-points2volume_SOURCES              = points2volume.cc
-santos_triang_SOURCES              = santos_triang.cc
-santos_dim4_triang_SOURCES         = santos_dim4_triang.cc
-santos_22_triang_SOURCES           = santos_22_triang.cc
+topcom_B_A_SOURCES                        = B_A.cc
+topcom_B_A_center_SOURCES                 = B_A_center.cc
+topcom_B_D_SOURCES                        = B_D.cc
+topcom_B_D_center_SOURCES                 = B_D_center.cc
+topcom_B_S_SOURCES                        = B_S.cc
+topcom_B_S_center_SOURCES                 = B_S_center.cc
+topcom_binomial_SOURCES                   = binomial.cc
+topcom_check_SOURCES                      = check.cc
+topcom_chiro2allfinetriangs_SOURCES       = chiro2allfinetriangs.cc
+topcom_chiro2alltriangs_SOURCES           = chiro2alltriangs.cc
+topcom_chiro2circuits_SOURCES             = chiro2circuits.cc
+topcom_chiro2cocircuits_SOURCES           = chiro2cocircuits.cc
+topcom_chiro2dual_SOURCES                 = chiro2dual.cc
+topcom_chiro2finetriang_SOURCES           = chiro2finetriang.cc
+topcom_chiro2finetriangs_SOURCES          = chiro2finetriangs.cc
+topcom_chiro2mintriang_SOURCES            = chiro2mintriang.cc
+topcom_chiro2nallfinetriangs_SOURCES      = chiro2nallfinetriangs.cc
+topcom_chiro2nalltriangs_SOURCES          = chiro2nalltriangs.cc
+topcom_chiro2ncircuits_SOURCES            = chiro2ncircuits.cc
+topcom_chiro2ncocircuits_SOURCES          = chiro2ncocircuits.cc
+topcom_chiro2nfinetriangs_SOURCES         = chiro2nfinetriangs.cc
+topcom_chiro2ntriangs_SOURCES             = chiro2ntriangs.cc
+topcom_chiro2placingtriang_SOURCES        = chiro2placingtriang.cc
+topcom_chiro2triangs_SOURCES              = chiro2triangs.cc
+topcom_cocircuits2facets_SOURCES          = cocircuits2facets.cc
+topcom_cross_SOURCES                      = cross.cc
+topcom_cube_SOURCES                       = cube.cc
+topcom_cyclic_SOURCES                     = cyclic.cc
+topcom_Dnxk_SOURCES                       = Dnxk.cc
+topcom_hypersimplex_SOURCES               = hypersimplex.cc
+topcom_lattice_SOURCES	                   = lattice.cc
+topcom_kDn_SOURCES                        = kDn.cc
+topcom_permutahedron_SOURCES              = permutahedron.cc
+topcom_points2allfinetriangs_SOURCES      = points2allfinetriangs.cc
+topcom_points2alltriangs_SOURCES          = points2alltriangs.cc
+topcom_points2chiro_SOURCES               = points2chiro.cc
+topcom_points2circuits_SOURCES            = points2circuits.cc
+topcom_points2cocircuits_SOURCES          = points2cocircuits.cc
+topcom_points2facets_SOURCES              = points2facets.cc
+topcom_points2finetriang_SOURCES          = points2finetriang.cc
+topcom_points2finetriangs_SOURCES         = points2finetriangs.cc
+topcom_points2flips_SOURCES               = points2flips.cc
+topcom_points2gale_SOURCES                = points2gale.cc
+topcom_points2mintriang_SOURCES           = points2mintriang.cc
+topcom_points2nallfinetriangs_SOURCES     = points2nallfinetriangs.cc
+topcom_points2nalltriangs_SOURCES         = points2nalltriangs.cc
+topcom_points2ncircuits_SOURCES           = points2ncircuits.cc
+topcom_points2ncocircuits_SOURCES         = points2ncocircuits.cc
+topcom_points2nfinetriangs_SOURCES        = points2nfinetriangs.cc
+topcom_points2nflips_SOURCES              = points2nflips.cc
+topcom_points2ntriangs_SOURCES            = points2ntriangs.cc
+topcom_points2placingtriang_SOURCES       = points2placingtriang.cc
+topcom_points2prettyprint_SOURCES         = points2prettyprint.cc
+topcom_points2symmetries_SOURCES          = points2symmetries.cc
+topcom_points2triangs_SOURCES             = points2triangs.cc
+topcom_points2vertices_SOURCES            = points2vertices.cc
+topcom_points2volume_SOURCES              = points2volume.cc
+topcom_santos_triang_SOURCES              = santos_triang.cc
+topcom_santos_dim4_triang_SOURCES         = santos_dim4_triang.cc
+topcom_santos_22_triang_SOURCES           = santos_22_triang.cc
 
 LDADD           = ../lib-src/libTOPCOM.a \
                   ../lib-src-reg/libCHECKREG.a
--- a/src-reg/Makefile.am
+++ b/src-reg/Makefile.am
@@ -1,6 +1,6 @@
-bin_PROGRAMS = checkregularity
+bin_PROGRAMS = topcom-checkregularity
 
-checkregularity_SOURCES = checkregularity.cc
+topcom_checkregularity_SOURCES = checkregularity.cc
 
 LDADD           = ../lib-src/libTOPCOM.a \
                   ../lib-src-reg/libCHECKREG.a \
